pkg/buildah/common.go

```diff
@@ -238,3 +241,36 @@ Flags:
 {{end}}
 `
 }
+
+func getContextDir(inputArgs []string) (string, error) {
```

Comment:
Since this function is only used by the `buildCmd` method, it is more appropriate to move it to the `build.go` file?